devices/milightRF24.coffee

```diff
@@ -0,0 +1,210 @@
+module.exports = (env) ->
+  Promise = env.require 'bluebird'
+  _ = require 'lodash'
+  Color = require 'color'
+  nodeMilightRF24 = require 'node-milight-rf24'
+  Buttons = nodeMilightRF24.RGBWButtons
+  nodeMilightRF24 =  nodeMilightRF24.MilightRF24Controller
+  BaseLedLight = require('./base')(env)
+  
+  class MilightRF24
+    constructor: (@config) ->
+      @gateway = new nodeMilightRF24({port: @config.MilightRF24Port})
+      env.logger.debug "Opening "+@config.MilightRF24Port
+      @gateway.open()
+
+      @gateway.on("Received", (data) ->
+        env.logger.debug data
+      )
+    
+    getGateway: ->
+      @gateway
+    
+    getDevice: (config, lastState) ->
+      new MilightRF24Zone(config, lastState, @)
+      
+    setColor: (id, zone, r,g,b) ->
+      env.logger.debug "Sending Color. Addr:"+id+" Zone:"+zone+" Red:"+r+" Green:"+g+" Blue:"+b
```

Comment:
Since it is written in CoffeeScript you can make use of the syntactic sugar it provides for string interpolation. So you could also write here: 'env.logger.debug "Sending Color. Addr: #{id}  Zone: #{zone}  Red: #{r} Green: #{g} Blue: #{b}"